modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-test/src/testFrontend/tests/dynamic-data-mapping/dynamic-data-mapping-type-text/text_field.js

```diff
@@ -169,7 +169,31 @@ describe(
 
 		describe(
 			'regression',
-			function() {}
+			function() {
+				it.only(
+					'shouldn\'t multi-line Text Field changes size on focus',
+					function(done) {
+						var textField = new Liferay.DDM.Field.Text(
+							{
+								context: {
+									name: 'textField',
+									displayStyle: 'multiline'
+								}
+							}
+						).render(document.body);
+
+						var textareaHeight = textField.getInputNode().get('offsetHeight');
+
+						textField.getInputNode().focus();
+
+						textField.syncInputHeight();
```

Comment:
Isn't the regression test only supposed to focus the input and check if the height doesn't change?